src/main/java/com/hida/Minter.java

```diff
@@ -0,0 +1,475 @@
+package com.hida;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+import org.codehaus.jackson.map.ObjectMapper;
+
+/**
+ * Minter class creates formatted ids in json using a variety of parameters.
+ *
+ * @author Brittany Cruz
+ * @author lruffin
+ */
+public class Minter {
+
+    private final static DatabaseManager DATABASE_MANAGER
+            = new DatabaseManager();
+    private final String CHAR_MAP;
+    private final HashMap<Character, String> BASE_MAP = new HashMap();

```

Comment:
The instantiation of the `HashMap` should include the type of the data (see similar comment in `ArkMinter.java`)